source/proof_resolution/spec/proof_resolution_spec.rb

```diff
@@ -145,6 +132,22 @@ def stub_ssm(key_values)
       end
     end
 
+    context 'when called with lexisnexis_configs in the lambda payload' do
+      let(:event) do
+        super().merge(
+          lexisnexis_config: {
+            username: 'aaa',
+            passwprd: 'bbb',
```

Comment:
typo although the config params are irrelevent

Reply:
better to fix now, thanks